src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java

```diff
@@ -166,84 +166,103 @@ public void collect(Collector collector) {
 
         clean(collector);
         List<GitHubRepo> enabledRepos = enabledRepos(collector);
+        LOG.info("GitHubCollectorTask:collect start, total enabledRepos=" + enabledRepos.size());
```

Comment:
instead of using .size() can we use CollectionUtils.size() to be null safe

Reply:
That is already done in enabledRepos(Collector collector): if (CollectionUtils.isEmpty(pulledRepos)) { return new ArrayList<>(); }

Reply:
Thanks